### PR TITLE
Extended Polycyclic Group docstrings

### PR DESCRIPTION
**Pull request by divyanshu132**
#### Release Notes

*  combinatorics
   * Extended polycyclic group docstrings

### CODE REVIEW

File: sympy/combinatorics/pc_groups.py [hunk 1]
```diff
@@ -10,6 +10,27 @@ class PolycyclicGroup(DefaultPrinting):
     is_solvable = True
 
     def __init__(self, pc_sequence, pc_series, relative_order, collector=None):
+        """
+
+        Parameters
+        ==========
+
+        pc_sequence
+                Polycyclic sequence is formed by collecting all the
```
Comment (jksuom) [inline sympy/combinatorics/pc_groups.py]: I think that the explanation of how the sequence and series are formed would be better suited to the docstring of where the sequence is constructed (in perm_groups). Here we could have, for pc_series, something like "A subnormal sequence of subgroups where each factor group is cyclic", and for pc_sequence, "A sequence of elements whose classes generate the cyclic factor groups of pc_series". Relative order could be "The orders of factor groups of pc_series". Also, each parameter has a type that will come on the same line as the name after "space, colon, space" like `pc_sequence : list`.

Comment (divyanshu132) [reply sympy/combinatorics/pc_groups.py]: Suppose we've a word, what should be it's type, may be `word : FreeGroupElement`?

Comment (jksuom) [reply sympy/combinatorics/pc_groups.py]: I think that is the proper type.

Comment (divyanshu132) [reply sympy/combinatorics/pc_groups.py]: Ok.

File: sympy/combinatorics/pc_groups.py [hunk 2]
```diff
@@ -33,28 +50,45 @@ class Collector(DefaultPrinting):
            Section 8.1.3
     """
 
-    def __init__(self, pcgs, pc_series, relative_order, group=None, pc_presentation=None):
+    def __init__(self, pcgs, pc_series, relative_order, FreeGroup=None, pc_presentation=None):
+        """
+
+        Most of the Parameters of Class Collector is same as PolycyclicGroup.
```
Comment (oscargus) [inline sympy/combinatorics/pc_groups.py]:
```suggestion
        Most of the parameters for the Collector class are the same as for PolycyclicGroup.
```

File: sympy/combinatorics/pc_groups.py [hunk 3]
```diff
@@ -33,28 +50,45 @@ class Collector(DefaultPrinting):
...
+        Parameters
+        ==========
+
+        FreeGroup : tuple
+                FreeGroup provides the mapping of polycyclic generating
+                sequence with the free group elements.
+        pc_presentation : dict
+                Provides the presentation of polycyclic groups with the
+                help of power and conjugate relators.
+
```
Comment (oscargus) [inline sympy/combinatorics/pc_groups.py]:
```suggestion

        See Also
        ========
        
        PolycyclicGroup
        
```

**Comment by oscargus** [comment]
Looks good!

Some suggestions. (If you are happy with them click "Add suggestion to batch" and at the end you can commit all in one go.)

**Comment by divyanshu132** [comment]
Thanks, Oscar for the suggestions.

File: sympy/combinatorics/pc_groups.py [hunk 4]
```diff
@@ -33,28 +50,50 @@ class Collector(DefaultPrinting):
            Section 8.1.3
     """
 
-    def __init__(self, pcgs, pc_series, relative_order, group=None, pc_presentation=None):
+    def __init__(self, pcgs, pc_series, relative_order, FreeGroup=None, pc_presentation=None):
```
Comment (jksuom) [inline sympy/combinatorics/pc_groups.py]: `FreeGroup` is the name of a class. Something like `free_group` would be better here.

Comment (divyanshu132) [reply sympy/combinatorics/pc_groups.py]: `free_group` was generating an error so I've used `free_group_`.

**Comment by jksuom** [comment]
Thanks, I think this is ready to go.

**Comment by divyanshu132** [comment]
Thanks!!

**Pull request merged**